grammars/ruby.cson

```diff
@@ -1940,6 +1952,10 @@
     'name': 'punctuation.separator.key-value'
   }
   {
+    'match': '->'
+    'name': 'keyword.operator.lambda.ruby'
```

Comment:
>  In Ruby the `->` is an operator that serves as a shortcut to the Kernel#lambda method

It should be tokenzied same as `lambda` keyword then (`support.function.kernel.arrow`)
Don't known why it is tokenized as `support.function`. As far as I understand, `lambda` and `->` in Ruby "define" methods/functions i.e. related to "storage" (`lambda` in python is `storage.type.function`)